src/util/libbacktrace.h

```diff
@@ -0,0 +1 @@
+../../libbacktrace/backtrace.h
```

Comment:
i think a better way is to have a proper `Findlibbacktrace.cmake` which exports `libbacktrace::libbacktrace`, and link `seasetar` against it if `Seastar_BACKTRACE` is set. yeah, we need a dedicated option for this feature.